### PR TITLE
Share [#]

### PR DESCRIPTION
**Pull request by pbanda1**
"Implementing share button functionality"

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,23 +17,25 @@
       <div class="title"></div>
       <div class="info">
         <button id="toggleMode" alt="toggle-switch">*</button>
-        <button id="shareButton" alt="share-theme">#</button>
+        <button id="shareButton" alt="share-theme" >#</button>
```
Comment (highskore) [inline index.html:20]: remove extra space

File: index.html [hunk 2]
```diff
@@ -17,23 +17,25 @@
...
+        <div id="copiedMessage" class="cloudMessage" class="cloud-message">Link copied!</div>
         <div class="world">world: <span id="datetimeInfo"></span></div>
         <div class="you">you: <span id="browserInfo"></span></div>
         <div class="me">
           me: a blockchain dev currently building smart contracts and backend at
           <a href="https://find.xyz/" target="_blank">find</a> and <a href="http://zeedz.io" target="_blank">zeedz</a>.
         </div>
         <div class="socials">
-          < <a href="http://twitter.com/0xhighskore" target="_blank">Twitter</a> |
```
Comment (highskore) [inline index.html:28]: return the "<"

File: index.html [hunk 3]
```diff
@@ -17,23 +17,25 @@
...
+           <a href="http://twitter.com/0xhighskore" target="_blank">Twitter</a> |
           <a href="http://github.com/highskore" target="_blank">GitHub</a> |
           <a href="http://medium.com/@lu_ka_ra_ch_ki" target="_blank">Medium</a> |
           <a href="mailto:me@highskore.com" target="_blank">Email</a> >
         </div>
       </div>
       <div class="buttons">
         <button id="restart">RESTART</button>
-        <button id="bio" onclick="window.location.href='bio'">BIO</button>
+        <button id="PR">PUBLIC RELATIONS</button>
```
Comment (highskore) [inline index.html:37]: remove "PR" button

File: index.html [hunk 4]
```diff
@@ -17,23 +17,25 @@
       <div class="title"></div>
       <div class="info">
         <button id="toggleMode" alt="toggle-switch">*</button>
-        <button id="shareButton" alt="share-theme">#</button>
+        <button id="shareButton" alt="share-theme" >#</button>
+        <div id="copiedMessage" class="cloudMessage" class="cloud-message">Link copied!</div>
```
[the remark this answers is not in the document]
Comment (pbanda1) [reply index.html:21]: +

File: index.html [hunk 5]
```diff
@@ -17,23 +17,25 @@
...
         <div class="world">world: <span id="datetimeInfo"></span></div>
         <div class="you">you: <span id="browserInfo"></span></div>
         <div class="me">
           me: a blockchain dev currently building smart contracts and backend at
           <a href="https://find.xyz/" target="_blank">find</a> and <a href="http://zeedz.io" target="_blank">zeedz</a>.
         </div>
         <div class="socials">
-          < <a href="http://twitter.com/0xhighskore" target="_blank">Twitter</a> |
+           < <a href="http://twitter.com/0xhighskore" target="_blank">Twitter</a> |
```
Comment (sztok7) [inline index.html:29]: also, it's safer to write `&lt;` instead of `<` to avoid this kind of situations

**Pull request merged**